src/logic/textBox.js

```diff
@@ -76,25 +76,19 @@ export const initTextBoxBehavior = () => {
     sendMessage("Tell me a fact").then((factResponse) => {
```

Comment:
you can change all these anonymus functions to one generic. 

See tat you always give response and run 3 functions. Maby instead do something like:

const handleResponse = response => {
      displayTextBox(response );
      addData("Joke", response );
      scrollDown();
} 

Then you will get stick to the DRY(Don't repeat yourself) code rule.